R/dist_quantiles.R

```diff
@@ -5,6 +5,9 @@ new_quantiles <- function(values = double(), quantile_levels = double()) {
   vec_cast(values, double())
   vec_cast(quantile_levels, double())
   stopifnot(length(values) == length(quantile_levels))
+  if (length(values) == 0L) {
+    return(distributional::dist_degenerate(double()))
+  }
```

Comment:
question: is this out of order vs. line 7?

question: why not a 0-length `dist_quantiles`?

Reply:
Right. I don't think we should ever hit this line. But I put it in anyway.